crates/vhost/src/vhost_user/message.rs

```diff
@@ -941,30 +941,30 @@ bitflags! {
     }
 }
 
-/// Max entries in one virtio-fs slave request.
+/// Max entries in one virtio-fs backend request.
 pub const VHOST_USER_FS_SLAVE_ENTRIES: usize = 8;
```

Comment:
What about updating also the name of this constant?
```suggestion
pub const VHOST_USER_FS_BACKEND_ENTRIES: usize = 8;
```

Reply:
Ah! Forgot about the all-uppercase case, I'll change this and check for other occurrences.